controls/radchartview/series/cartesianchart-series/area-series/rangesplineseries.md

```diff
@@ -0,0 +1,108 @@
+---
+title: RangeSplineSeries
+page_title: RangeSplineSeries
+description: RangeSplineSeries
+slug: radchartview-series-areaseries-rangesplineseries
+tags: rangesplineseries
+published: True
+position: 5
+---
+
+# RangeSplineSeries
+
+The __RangeSplineSeries__ behaves like an [SplineAreaSeries]({%slug radchartview-series-splineareaseries%}) but you can specify a minimum and maximum for each data point. By default the colors of the line and the area are the same.
+
+* [Declaratively Defined Series](#declaratively-defined-series)
+* [Properties](#properties)
+* [Data Binding](#data-binding)
+* [Styling the Series](#styling-the-series)
+
+## Declaratively Defined Series
+
+You can use the definition from __Example 1__ to display a RangeSplineSeries:
+
+#### __[XAML] Example 1: Declaring a RangeSplineSeries in XAML__
+{{region radchartview-series-areaseries-rangesplineseries_0}}
+	<telerik:RadCartesianChart Palette="Windows8">            
+		<telerik:RadCartesianChart.HorizontalAxis>
+			<telerik:CategoricalAxis/>
+		</telerik:RadCartesianChart.HorizontalAxis>            
+		<telerik:RadCartesianChart.VerticalAxis>
+			<telerik:LinearAxis/>
+		</telerik:RadCartesianChart.VerticalAxis>
+		<telerik:RadCartesianChart.Series>
+			<telerik:RangeSplineSeries>
+				<telerik:RangeSplineSeries.DataPoints>
+					<telerik:RangeDataPoint Category="January" High="5" Low="0"/>
+					<telerik:RangeDataPoint Category="February" High="7" Low="2"/>
+					<telerik:RangeDataPoint Category="March" High="6" Low="4"/>
+					<telerik:RangeDataPoint Category="April" High="8" Low="5"/>
+					<telerik:RangeDataPoint Category="May" High="4" Low="3"/>
+					<telerik:RangeDataPoint Category="June" High="9" Low="7"/>
+					<telerik:RangeDataPoint Category="July" High="3" Low="1"/>
+				</telerik:RangeSplineSeries.DataPoints>
+			</telerik:RangeSplineSeries>
+		</telerik:RadCartesianChart.Series>
+	</telerik:RadCartesianChart>
+{{endregion}}
+
+#### __Figure 1: RangeSplineSeries visual appearance__  
+![radchartview-series-areaseries-rangesplineseries](images/radchartview-series-rangesplineseries.png)
+
+## Properties
+
+* __CategoryBinding__: A property of type __DataPointBinding__ that gets or sets the property path that determines the category value of the data point.
+* __HighBinding__: A property of type __DataPointBinding__ that gets or sets the property path that determines the high value of the data point.
+* __LowBinding__: A property of type __DataPointBinding__ that gets or sets the property path that determines the low value of the data point.
+* __Fill__: A property of type __Brush__ that gets or sets the color of the RangeSplineSeries area.
+* __DashArray__: A property of type __DoubleCollection__ that gets or sets the dash pattern applied to the stroke of the area.
+* __Stroke__: A property of type __Brush__ that gets or sets the outline stroke of the RangeSplineSeries area.
```

Comment:
You can add one more sentence in the Stroke property bullet that tells there is a StrokeThickness prop exposed by the series.